src/redis.nim

```diff
@@ -246,7 +249,10 @@ proc readSingleString(
 proc readSingleString(r: Redis | AsyncRedis): Future[RedisString] {.multisync.} =
   # TODO: Rename these style of procedures to `processSingleString`?
   let line = await r.managedRecvLine()
-  if line == nil:
+
+  echo "log line", line
```

Comment:
Looks like this debug code should be removed?

```suggestion
```

Reply:
oops